### ISSUE TITLE
fresh start virtualbox: kube-apiserver: timed out waiting for the condition

### ISSUE THREAD

**Issue by danchirillo** [kubernetes/minikube#4045]
OS:  MacOS 10.14.4

Command Line Output:
○ → minikube start
😄  minikube v1.0.0 on darwin (amd64)
👍  minikube will upgrade the local cluster from Kubernetes 1.13.4 to 1.14.0
🤹  Downloading Kubernetes v1.14.0 images in the background ...
💡  Tip: Use 'minikube start -p <name>' to create a new cluster, or 'minikube delete' to delete this one.
🔄  Restarting existing virtualbox VM for "minikube" ...
⌛  Waiting for SSH access ...
📶  "minikube" IP address is 192.168.99.100
🐳  Configuring Docker as the container runtime ...
🐳  Version of container runtime is 18.06.2-ce
⌛  Waiting for image downloads to complete ...
E0403 09:20:38.158620   28447 start.go:209] Error caching images:  Caching images for kubeadm: caching images: caching image /Users/admin/.minikube/cache/images/k8s.gcr.io/etcd_3.3.10: stream error: stream ID 35; INTERNAL_ERROR
✨  Preparing Kubernetes environment ...
❌  Unable to load cached images: loading cached images: loading image /Users/admin/.minikube/cache/images/k8s.gcr.io/etcd_3.3.10: stat /Users/admin/.minikube/cache/images/k8s.gcr.io/etcd_3.3.10: no such file or directory
💾  Downloading kubeadm v1.14.0
💾  Downloading kubelet v1.14.0

🚜  Pulling images required by Kubernetes v1.14.0 ...
🔄  Relaunching Kubernetes v1.14.0 using kubeadm ...
⌛  Waiting for pods: apiserver

💣  Error restarting cluster: wait: waiting for component=kube-apiserver: timed out waiting for the condition

😿  Sorry that minikube crashed. If this was unexpected, we would love to hear from you:
👉  https://github.com/kubernetes/minikube/issues/new
❌  Problems detected in "kube-addon-manager":
    error: no objects passed to apply
    error: no objects passed to apply
    error: no objects passed to apply

**Comment by danchirillo** [comment]
[klog.log](https://github.com/kubernetes/minikube/files/3038992/klog.log)

**Comment by balopat** [comment]
Thank you for opening this!  Can you share what addons you have turned on?

**Comment by balopat** [comment]
More info on #4036.

**Comment by tstromberg** [comment]
This might have been an upgrade gone bad. Does `minikube delete` workaround the problem by any chance?

**Comment by tdensmore** [comment]
`minikube delete` followed by `minikube start` worked for me.

**Comment by Tyagi-Gaurav** [comment]
Is there any other fix for this issue? I get this error while running minikube on mac OS High Sierra 10.13.6. 
```
  Pulling images required by Kubernetes v1.14.0 ...
🚀  Launching Kubernetes v1.14.0 using kubeadm ...
⌛  Waiting for pods: apiserver
💣  Error starting cluster: wait: waiting for component=kube-apiserver: timed out waiting for the condition
```

Even `delete` followed by `start` does not help. I even tried deleting `~/.miniKube` folder before `start` but that did not change anything.

**Comment by tstromberg** [comment]
I believe this issue was resolved in the v1.1.0 release. Please try upgrading to the latest release of minikube and run `minikube delete` to remove the previous cluster state.

If the same issue occurs, please re-open this bug. Thank you opening this bug report, and for your patience!
